Proposals/0001-bas-broek.md

```diff
@@ -0,0 +1,49 @@
+# Scaling A Codeless Open Source Swift Community with Bas Broek
+
+* Proposal: [will add PR#](will add PR link)
+
+* Authors: [Garric Nahapetian](https://github.com//garricn)
+
+* Review Manager: [will add name](will add profile link)
```

Comment:
I need a review manager @JohnSundell @lattner

Reply:
I can do this (though I'm not sure what is involved :-).